pkgs/development/compilers/opensycl/default.nix

```
@@ -0,0 +1,49 @@
+{ lib, clang15Stdenv, fetchFromGitHub, llvmPackages_15, lld_15, llvmPackages_rocm, rocm-device-libs, python3, rocm-runtime, cmake, boost, libxml2, libffi, makeWrapper, hip, rocmSupport ? false }:
```

this should be multiple lines like this:
```suggestion
{ lib
, clang15Stdenv
, fetchFromGitHub
, llvmPackages_15
, lld_15
, llvmPackages_rocm
, rocm-device-libs
, python3
, rocm-runtime
, cmake
, boost
, libxml2
, libffi
, makeWrapper
, hip
, rocmSupport ? false 
}:
```

---

Done and looks much nicer this way